spec/models/recruitment_cycle_spec.rb

```diff
@@ -13,7 +13,7 @@
 require 'rails_helper'
 
 describe RecruitmentCycle, type: :model do
-  subject { create(:recruitment_cycle, year: "2019") }
+  subject { RecruitmentCycle.find_by(year: "2019") }
```

Comment:
There's already a 2019 one; `creating` one as we did before would result in **two** 2019 recruitment cycles.